content/en/docs/setup/install/multicluster/shared-gateways/index.md

```diff
@@ -180,7 +180,6 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
     $ CLUSTER_NAME=$(kubectl --context=$CTX_CLUSTER2 config view --minify=true -o jsonpath='{.clusters[].name}')
     $ istioctl manifest apply --context=$CTX_CLUSTER2 \
       --set profile=remote \
-      --set values.global.mtls.enabled=true \
```

Comment:
Does the removal on this line mean that global.mtls.enabled is deprecated? Has the related code been removed from the Istio implementation?

Reply:
We still have that flag, but don't want users to continue to use that. We will remove it in future release.